src/Couchbase.Lite.Shared/API/Query/Meta.cs

```diff
@@ -53,6 +53,16 @@ public static class Meta
         [NotNull]
         public static IMetaExpression Sequence => new QueryTypeExpression(SequenceKeyPath, SequenceColumnName);
 
+        /// <summary>
+        /// A metadata expression refering to the deleted boolean flag of the document.
+        /// </summary>
+        public static IMetaExpression IsDeleted => new QueryTypeExpression(SequenceKeyPath, SequenceColumnName);
+
+        /// <summary>
+        /// A metadata expression refering to the expiration date of the document.
+        /// </summary>
+        public static IMetaExpression Expiration => new QueryTypeExpression(SequenceKeyPath, SequenceColumnName);
```

Comment:
Ditto for this one.

Reply:
I'm not sure if LiteCore has defined this or not.

Reply:
LiteCore just updated: https://github.com/couchbase/couchbase-lite-core/wiki/JSON-Query-Schema#magic-metadata-properties _expiration is correct.